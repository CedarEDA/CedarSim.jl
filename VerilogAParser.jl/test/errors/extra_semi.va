module extra_semi(p, n);
    analog function real expclamp;
        input x;

        begin;
            if (x > `EXP_THRESHOLD)
                expclamp = `MAX_EXP;
            else if (x < `EXP_THRESHOLD)
                expclamp = `MIN_EXP;
            else
                expclamp = exp(x);
        end
    endfunction
endmodule