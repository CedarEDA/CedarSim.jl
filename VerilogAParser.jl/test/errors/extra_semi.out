[31m[1mâ•­ ERROR [VAS46]:[22m[39m[31m Unexpected semicolon.[39m
[31mâ”‚[39m[90m   â•­â”€â”€â”€â”€â”€ extra_semi.va:5[39m
[31mâ”‚[90m5[0m  [90mâ”‚[0m         begin[91m;
[31mâ”‚[0m   [90mâ”Š[0m              [91mâ”“
[31mâ”‚[0m   [90mâ”Š[0m   [91mâœ— Unexpected semicolon[0m
[31mâ”‚[39m[90m   â”‚[39m
[31mâ”‚[39m[90m   â•°â”€â”€â”€â”€â”€ [39mUnexpected semicolon.
